src/components/CourseRating.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./components styles/CourseRating.css";
+
+function CourseRating(props) {
+  return (
+    <div className="rating-box">
+      <div className="rate">{props.rate}.0</div>
+      <div className="stars">
+        <div className="star"></div>
```

Comment:
you can make it like this also which is the good practice
`<div className="star"/>`